packages/card/src/react/index.js

```diff
@@ -105,8 +99,16 @@ const ActionButton = glamorous.button(
         }
       : null
 )
+
+const ActionBarAction = props => {
+  const filteredProps = { title: props.title }
+  if (props.onClick) filteredProps.onClick = props.onClick
+  return <ActionButton {...filteredProps}>{props.icon}</ActionButton>
+}
+
 ActionBarAction.propTypes = {
-  icon: PropTypes.element.isRequired
+  icon: PropTypes.element.isRequired,
+  title: PropTypes.string.isRequired
```

Comment:
k, we're going with required, people will get this minor upgrade automatically in many cases and start to get this error.  That's the plan, right?  I'm fine with that.

Might be worth putting that fact in the commit message so it shows up in the changelog.  Well, I guess the title will do that.